.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishMergedExtractorConfiguration.cs

```diff
@@ -96,6 +129,39 @@ public EnglishMergedExtractorConfiguration(IDateTimeOptionsConfiguration config)
                 SinceRegex = SinceRegexExp;
             }
 
+            if ((config.Options & DateTimeOptions.TasksMode) != 0)
+            {
+                TasksModeMentionFilters = new List<Regex>
```

Comment:
Please move this init into it's own protected method and fix code indentation. It's easier to read the constructor code.